specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-02-01/managedClusters.json

```diff
@@ -1652,6 +1652,20 @@
           },
           "description": "Network policy used for building Kubernetes network."
         },
+        "networkMode": {
+          "type": "string",
+          "enum": [
+            "",
```

Comment:
>"" [](start = 12, length = 2)

What does empty string mean here?

Reply:
@majastrz, this means that the networkMode is not set.

Reply:
Can we use a non-empty value to represent that? Something like `none` or `default` or 'NotSet`?

Reply:
Discussed over IM. The API is apparently already released, which violates the usual review workflow (swagger-first)

Reply:
thank you, will follow that.

Reply:
This is breaking C# generated code, see https://github.com/AzureSDKAutomation/azure-sdk-for-net/pull/418/files#diff-2f25f07072c1b2f687283e3233db4dc6R19